cmd/posttriage/checks.go

```diff
@@ -35,17 +35,9 @@ func ReleaseBlockerFromIssue(issue jira.Issue) (releaseBlocker, error) {
 		return releaseBlockerNone, nil
 	}
 
-	releaseBlockerSlice, err := issue.Fields.Unknowns.Slice("customfield_12319743")
-	if err != nil {
-		return releaseBlockerNone, err
-	}
-	if len(releaseBlockerSlice) < 1 {
-		return releaseBlockerNone, nil
-	}
-
-	releaseBlockerMap, ok := releaseBlockerSlice[0].(map[string]any)
+	releaseBlockerMap, ok := issue.Fields.Unknowns["customfield_12319743"].(map[string]any)
 	if !ok {
-		return releaseBlockerNone, fmt.Errorf("failed to parse Test Coverage")
+		return releaseBlockerNone, fmt.Errorf("failed to parse (not a map)")
```

Comment:
Should we still say what field we failed to parse (but, you know, name the correct field this time :sweat_smile:). Ditto for below.

Reply:
The function is called from “func releaseBlockerCheck” that itself already wraps the error in:
fmt.Errorf("failed to parse Release Blocker: %s", err)

The same should be true for testCoverage

Reply:
ack